marketplace-and-integrations/README.md

```diff
@@ -32,6 +32,10 @@ Umbraco makes available four licensed add-ons that are available for purchase. T
 
 <table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>Umbraco Forms</strong></td><td>Extend your Umbraco CMS website with custom forms in all shapes and sizes.</td><td><a href="https://docs.umbraco.com/umbraco-forms">https://docs.umbraco.com/umbraco-forms</a></td><td><a href=".gitbook/assets/forms7.png">forms7.png</a></td></tr><tr><td><strong>Umbraco Deploy</strong></td><td>Synchronize content and code changes between your Umbraco website environments.</td><td><a href="https://docs.umbraco.com/umbraco-deploy">https://docs.umbraco.com/umbraco-deploy</a></td><td><a href=".gitbook/assets/paperplane7.png">paperplane7.png</a></td></tr><tr><td><strong>Umbraco Workflow</strong></td><td>Structure content reviews and approval flows for Content Editors on your website.</td><td><a href="https://docs.umbraco.com/umbraco-workflow">https://docs.umbraco.com/umbraco-workflow</a></td><td><a href=".gitbook/assets/Workflow7.png">Workflow7.png</a></td></tr><tr><td><strong>Umbraco Commerce</strong></td><td>Setup shop within a familiar structure and interface that is the Umbraco CMS with Umbraco Commerce.</td><td><a href="https://docs.umbraco.com/umbraco-commerce">https://docs.umbraco.com/umbraco-commerce</a></td><td><a href=".gitbook/assets/umbraco-commerce-logo.png">umbraco-commerce-logo.png</a></td></tr></tbody></table>
 
+We aim to make all commercial products available as new major versions in alignment with the releases of Umbraco CMS. A new major version of Umbraco, running on the latest version of the .NET framework, is expected every six months. At or around the same time, compatible Forms, Deploy, Workflow and Commerce releases will also be made available, along with [the additional, dependent packages](./commercial-product-dependencies.md).
```

Comment:
AFAIK [.net frameworks ](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework) are not same as what v10+ use it is just .net. I also added a link to it.
```suggestion
We aim to make all commercial products available as new major versions in alignment with the releases of Umbraco CMS. A new major version of Umbraco, running on the latest version of [.NET](https://dotnet.microsoft.com/en-us/download/dotnet), is expected every six months. At or around the same time, compatible Forms, Deploy, Workflow and Commerce releases will also be made available, along with [the additional, dependent packages](./commercial-product-dependencies.md).
```